xgi/drawing/xgi_pylab.py

```diff
@@ -54,29 +56,45 @@ def draw(
     ax : matplotlib.pyplot.axes (default=None)
```

Comment:
Just above: cmap is described in the docstring but it is not an argument of draw() anymore

Reply:
good call!